content/05.main-text.md

```diff
@@ -257,8 +257,51 @@ We converted Hetionet v1.0 to HetMat format and uploaded the `hetionet-v1.0.hetm
 
 ### Permuted hetnets
 
+In order to generate a null distribution for a DWPC, we rely on DWPCs computed from permuted hetnets.
+We derive permuted hetnets from the unpermuted network using the XSwap algorithm [@10.1137/1.9781611972795.67].
+XSwap randomizes edges while preserving node degree.
+Therefore, it's ideal for generating null distributions that retain general degree effects,
+but destroy the actual meaning of edges.
+We adapt XSwap to hetnets by applying it separately to each metaedge [@rephetio; @doi:10.15363/thinklab.d178; @xswap].
+
+Project Rephetio created 5 permuted hetnets [@rephetio; @doi:10.15363/thinklab.d178],
+which were used to generate a null distribution of classifier performance for each metapath-based feature.
+Here, we aim to create a null distribution for individual DWPCs, which requires vastly more permuted values to estimate with accuracy.
+Therefore, we generated 200 permuted hetnets ([archive](https://github.com/greenelab/hetmech/blob/042063fb559048e52b3dc2731b6d6c6836f698cf/data/hetionet-v1.0-permutations.zip)).
```

Comment:
@zietzm I forget which code we used to generate these 200 permutations (xswap or old school hetio code).

Looking at https://github.com/greenelab/hetmech/pull/127 it's not clear to me. @zietzm do you remember?

Reply:
This was definitely before the C/C++ implementation

Reply:
okay rephrased the xswap sentence in https://github.com/greenelab/connectivity-search-manuscript/pull/10/commits/bc8e49cb8b2ea2be1e1f7e5f9e7f5ce1064ea897